engine/src/main/scala/cromwell/webservice/ApiDataModels.scala

```diff
@@ -36,7 +36,7 @@ case class WorkflowMetadataResponse(id: String,
                                     calls: Map[String, Seq[CallMetadata]],
                                     failures: Option[Seq[FailureEventEntry]])
 
-case class WorkflowQueryResponse(results: Seq[WorkflowQueryResult])
+case class WorkflowQueryResponse(results: Seq[WorkflowQueryResult], page: Option[Int], pageSize: Option[Int], totalRecords: Int)
```

Comment:
@jacarey, I think by not making `totalRecords` an `Option[Int]`, can be a potentially breaking change.

At least it broke our very brittle front end :/. -- Just wanted to draw attention that you now always return totalRecords, even if pagination info is not requested.